Chapters/2-1-Chapter8.md

```diff
@@ -0,0 +1,159 @@
+
+Kipnu listened closely for signs of life as he attempted to quiet his rustling footsteps. This was not a place for people. There were no shrubs to forage, no fruiting clusters in sight, no animals scurrying up and down trees or whistling upon the air. It was unsettlingly silent. He cleared a patch of ground looking for anything that might have fallen, but only roots and compacted leaves appeared.
+
+"There was smoke when we came in. That must be from the other exiles. They must have food somehow," he said to himself. His understanding of Ulukoma was hazy at best but he set out in the direction of Ulu, where he presumed the center of the island to be.
+
+There was a small clearing in the trees where he noticed a peculiar ornament on one of them. It was made of thin rings of copper, forming a spiral, and oxidized to a sea green patina by the salt water in the air. On the top end of the spiral, the metal straightened out and entered the tree's trunk. It looked to be some strange tool that he couldn't fathom. He stopped to study them, running his hands and fingers against the wires, imagining all the strange things it might be used for.
+
+"Are you lost?" A voice suddenly called out.
+
+He turned around to find the child following his footsteps.
+
+"You've been following me?" Kipnu asked.
+
+"Yes," thi said, "They know you're here."
+
+"Who?"
+
+"The other exiles. They know the schedule. They wait for you to starve and then come to collect you when you're weak."
+
+"How do you know all this?"
+
+"This is my home."
+
+Kipnu was aghast imagining what this child's life might be like.
+
+"Where is your mother?" he asked.
+
+"My mother is waiting."
+
+"Waiting for what?"
+
+Kipnu looked into Linyu's eyes for answers, then noticed that the kaln had been staring wide-eyed at something behind him. Kipnu flung himself around with his torch held high and caught the nose of a woman dressed in a thick robe of black kelp, her skin covered with blue luminescent flow tattoos. In her left hand was a wooden spear as tall as she was. She grunted with fury at the torch, and removed it from Kipnu's hand with ease. The two began to run, but the dark made it hard to see and Kipnu tripped on a root after only a few steps. The woman ran after Linyu, but in a desparate effort to save hae, Kipnu grabbed the woman's legs. She stumbled to the ground along with the torch and her spear. Her body twisted itself, wrestling free from Kipnu's grasp in an instant, and she jumped forward to search for the child but the darkness and the leaves had veiled hae path. The mysterious woman stood frustrated for a few moments, then turned and ambled toward Kipnu with a furious look on her face, crushing the the torch under her bare foot.
```

Comment:
Action sequences should be broken up. Short sentences. Burst phrases. Simple.

The reader's velocity as they scan across the page should be faster than what is normal for the text, while a lot is happening.

Keep the words/events ratio low.